### DISCUSSION TITLE
Using Comfy with a cloud GPU?

### DISCUSSION THREAD

**Discussion by 1001celeritas** [comfyanonymous/ComfyUI#3065] [Q&A] [answered]
Hello, I've been using Comfy most days for a while now, my GPU went pop the other day and, I was wondering is it possible to connect the UI to an online GPU service maybe in the cloud?

Many thanks

**Comment by kadirnar** [comment, marked as answer]
https://huggingface.co/spaces/ArtGAN/Diffusion-API

**Reply by 1001celeritas** [reply]
kadirnar, thank you so much sir!
:)

**Comment by baddth** [comment]
If you’re looking for ComfyUI on the go, [ComfyAI](https://comfyai.run/) is worth a look. You can run it online, and they’ve got free cloud GPUs. Here’s a link with more details: [ComfyAI Blog](https://comfyai.run/blog/post/comfyui-online-cloud-free-serverless-hosting).
